justfile

```diff
@@ -256,21 +256,21 @@ bootstrap *ARGS:
 # --------------------------------------------------
 
 @tailwind *ARGS:
-    npx tailwindcss \
-        --config ./static/js/tailwind.config.js \
-        --input ./static/css/tailwindcss.css \
-        --output ./static/css/tailwindcss.min.css \
-        {{ ARGS }}
+    docker compose run django tailwind \
+    -i tailwind.css \
+    -o static/css/tailwind-output.css \
+    {{ ARGS }}
 
+# minify and optimize the output css
 @tailwind-build:
-    just tailwind build
+    just tailwind --minify
 
 @tailwind-lint:
     npx rustywind --check-formatted templates/
     # npx rustywind --write templates/
 
+
 @tailwind-watch:
-    just tailwind-build
     just tailwind --watch
```

Comment:
This should pass in the `watch` argument for django-tailwind-cli, I think.

Reply:
Pending clarification on https://github.com/djangopackages/djangopackages/pull/1308/files#r2052702659